### ISSUE TITLE
what is the specific  algorithm or technique is used ? can anyone help me

### ISSUE THREAD

**Issue by AlbandryNasser** [ageitgey/face_recognition#1036]
* face_recognition version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.
IMPORTANT: If your issue is related to a specific picture, include it so others can reproduce the issue.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Naeempatel010** [comment]
Yes, this implements the original paper on FaceNet by Google. But the only problem I feel is it doesn't reach the result as proposed in the paper(>99%)
